packages/@ourworldindata/grapher/src/schema/grapher-schema.008.yaml

```diff
@@ -217,17 +217,26 @@ properties:
         default: true
         description: Whether to hide the relative mode UI toggle
     comparisonLines:
-        description: List of vertical comparison lines to draw
+        description: List of comparison lines to draw
         type: array
         default: []
         items:
-            type: object
-            properties:
-                label:
-                    type: string
-                yEquals:
-                    type: string
-            additionalProperties: false
+            anyOf:
+                - type: object
+                  properties:
+                      label:
+                          type: string
+                      yEquals:
+                          type: string
+                  additionalProperties: false
+                - type: object
+                  properties:
+                      label:
+                          type: string
+                      xEquals:
+                          type: number
+                  required: [xEquals]
```

Comment:
ooc, what's `required`doing here? And why does the `yEquals` object not have the same?

Reply:
`yEquals` defaults to `x` if not provided (not my choice, that’s just how it works right now). For `xEquals`, defaulting to zero (or another constant) felt a bit arbitrary, so I figured it’s better to make it required. That said, I get how it might be confusing since `yEquals` behaves differently. Maybe I should add a note in the description at the very least.